server/src/main/java/models/Address.java

```diff
@@ -0,0 +1,24 @@
+package models;
+
+public class Address {
```

Comment:
Is this class for the address of the ground station? I would also add country (one of our ground stations is in Rome)

Reply:
Sorry about that! I made that class just to test out creating a document to upload to MongoDB. I'll delete that from the models package.